lib/models/note.js

```diff
@@ -23,7 +23,7 @@ var logger = require("../logger.js");
 var ot = require("../ot/index.js");
 
 // permission types
-var permissionTypes = ["freely", "editable", "locked", "private"];
+var permissionTypes = ["freely", "editable", "locked", "private", "limited", "protected"];
```

Comment:
Would prefer the order from loose to tight, like:
`freely`, `editable`, `limited`, `private`, `protected`, `locked`